### ISSUE TITLE
Acardian preview text contrast

### ISSUE THREAD

**Issue by helms-charity** [aemsites/ams-lts#53]
The gray preview text is failing accessibility, probably can make it black...
https://pagespeed.web.dev/analysis/https-49-bring-in-spectrum--ams-lts--aemsites-aem-page/qdt5hhjwvb?utm_source=psi&utm_medium=redirect&form_factor=mobile
